meetings/2021-07/july-14.md

```diff
@@ -0,0 +1,1028 @@
+
+AKI: Yeah, I agree But Bradford your reply.
+
+BSH: I just would like, as a Counterpoint to say that actually, it's misleading. I would think because people see these words from the outside and they make an assumption that they know what it means. If you say stage 0, they know they need to look up what the heck that means. And besides, I feel like since the past year I've been coming to this, everyone says State 0 proposal. stage 1 proposal, stage 2 proposal, stage 3... Nobody ever - I almost ever hear any of these other words so it would reflect what we actually do. If you just say they're all proposals with these stages.
+
+SYG: As another data point, when I have taught the stages in giving talks and in panels and stuff - I mean, I guess we can actually look this up because the panel's we have done in the past, we have videos, right? I don't think we have ever used the actual English words. We say the stages and then we kind of say a little sentence about what it generally means for the proposal to be at that stage. So I feel like when we do the teaching, when we do the evangelizing of the process, that we actually don't use the English words. And I want to highlight something that Hax said in the Matrix, which is Chinese Community are not English speakers, they're not going to latch onto these words, they're going to say stage 1 to stage 4.
+
+JHX: The English words like "proposal" (etc) are a problem, because if we use these words we need to translate them, and actually different people have very different ideas of what these words means. So we I think most of us are in China only use "stage 1" or 2, 3, just like that.
+
+HHM: So the general observation here, as most of them are comfortable, calling it by State 0 to stage 4, or rather than names, and the contrasting opinion is we have names, then it would be easier for people to explain it to newcomers, or folks who don't have idea about stages. The other opinion is like, if we call it stage zero, then they will probably go and look look into what "stage 0" means. So, I think the majority of folks are comfortable calling it stage 0 through stage 4 rather than by names. So should we get rid of names?
+
+SFC: I have two queue entries. First is, a Stage 0 proposal is not really formalized until it reaches Stage 1; it's not really even a proposal, it's almost more like a suggestion, because it's not really a proposal until it actually reaches Stage 1. So like if we were to keep the English names, I feel like Stage 1 being called "concept", Stage 0 being called something like "suggestion" or something like that.
+
+JHD: It's a proposal when a TC39 delegate champions it and plans to present it, that's usually the bar we use and that has been useful for things that did not make it to stage one, because they have come back again and they're still useful as reference points. So I think it's still useful to list stage 0.
```

Comment:
```suggestion
JHD: It's a proposal when a TC39 delegate champions it and plans to present it, that's usually the bar we use and that has been useful for things that did not make it to stage 1, because they have come back again and they're still useful as reference points. So I think it's still useful to list stage 0.
```